libs/perun/components/src/lib/password-reset/password-reset.component.html

```diff
@@ -1,20 +1,20 @@
-<h1 class="page-title">{{'PASSWORD_RESET.TITLE' | customTranslate | translate}}</h1>
+<h1 class="page-title">{{'SHARED_LIB.PERUN.COMPONENTS.PASSWORD_RESET.TITLE' | customTranslate | translate}}</h1>
```

Comment:
Also is here neccessarry to have there these double pipes for customTranslate and translate?

Reply:
At the moment yes, mainly because of the user-profile app where is this component used.